hub/powertoys/powerrename.md

```diff
@@ -110,6 +110,21 @@ A _Replace with_ text `Image_${padding=4;increment=2;start=10}_` would produce t
 - Image_0012_b.jpg
 - Image_0014_bc.jpg
 
+### Random string values
+
+If selected, you can use the following patterns as part of the _Replace with_ text:
+
+| Variable pattern    | Explanation                                                                                |
+| :------------------ | :----------------------------------------------------------------------------------------- |
+| `${rstringalnum=X}` | Random string with uppercase letters, lowercase letters and 0-9 digits, customized length. |
+| `${rstringalpha=X}` | Random string with uppercase letters and lowercase letters, customized length.             |
+| `${rstringdigit=X}` | Random string with 0-9 digits, customized length.                                          |
+| `${ruuidv4}`      | Random UUID according to v4 specification.                                                 |
+
+By default, random string values created are mixed case. You can adjust the generating behavior with the general [text formatting options that PowerRename provides](#text-formatting).
```

Comment:
Hm, the word "[case](https://en.wiktionary.org/wiki/case)", by itself, can have a number of very different meanings. Is there a word that would be more specific?

Reply:
Saying `mixed letter case` could tie it more closely to the context

Reply:
@Jay-o-Way / @jhirvioja - This is released on 0.82.0. I'm going to merge this now with the current wording. Feel free to circle back with another PR if you think of a better way to phrase the info about mixed case strings.